required/firstaid/latex2e-first-aid-for-external-files.dtx

```diff
@@ -740,12 +740,16 @@
 %    \end{macrocode}
 %  we test if \cs{@currentcounter} is empty for unnumbered sections
 % \changes{v1.1h}{2024/11/01}{fix spurious space}
+% \changes{v1.1j}{2024/11/24}{add \cs{iftag@} test, issue \#1560}
 %    \begin{macrocode}
+       \newif\iftag@
```

Comment:
I'm surprised that \newif doesn't bark, when the conditional aready exists, but it looks like that in fact the case. Not sure how god that is, but then it must have been that way forever.

Reply:
Yes, it comes from plain and plain doesn't check anything :)